internal/conf/configuration.go

```diff
@@ -799,6 +827,12 @@ func (config *GlobalConfiguration) ApplyDefaults() error {
 	if config.MFA.FactorExpiryDuration < defaultFactorExpiryDuration {
 		config.MFA.FactorExpiryDuration = defaultFactorExpiryDuration
 	}
+
+	if config.MFA.SMS.OtpLength == 0 || config.MFA.SMS.OtpLength < 6 || config.MFA.SMS.OtpLength > 10 {
+		// 6-digit otp by default
+		config.MFA.SMS.OtpLength = 6
+	}
```

Comment:
hmmm why do we need to check for 0 if we already check for < 6?
```suggestion
	if config.MFA.SMS.OtpLength < 6 || config.MFA.SMS.OtpLength > 10 {
		// 6-digit otp by default
		config.MFA.SMS.OtpLength = 6
	}
```

Reply:
Good catch, I don't think we need to check for 0. Lifted this from [the OTP configuration](https://github.com/supabase/auth/blob/master/internal/conf/configuration.go#L751) w/o checking.

Probably don't need to adjust that though or can be adjusted later